components/customs/arboStats.tsx

```diff
@@ -0,0 +1,21 @@
+"use client"
+
+import { useArboDataStatistics } from "@/hooks/useArboStatistics";
+
+export function formatNumber(num?: number) {
+    return (
+      <b className="px-1">{num?.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,') ?? 0}</b>
+      );
+}
+
+function ArboStats() {
+    const arboDataStats = useArboDataStatistics();
+
+    return (
+        <h3 className="flex text-background bg-white rounded-md px-16 justify-center p-8 whitespace-nowrap">
+          We have data from {formatNumber(arboDataStats.data?.arbovirusDataStatistics.patricipantCount)} Participants accross {formatNumber(arboDataStats.data?.arbovirusDataStatistics.estimateCount)} Estimates from {formatNumber(arboDataStats.data?.arbovirusDataStatistics.sourceCount)} Sources spanning {formatNumber(arboDataStats.data?.arbovirusDataStatistics.countryCount)} countries and territories
```

Comment:
Or just a combined stat for both something like: "We have all this data across all our dashboards. Explore it further" 

Since the main purpose of this is to be like SO MUCH DATA WHAT A WOW COME USE OUR PLATFORM

Reply:
I would prefer to make that change later as at the moment it really is specifically arboStats, when we change it in the future we can change the name! (just a personal preference so we know ++ maybe we want to show these stats on arbo/sc2treacker respectively and may want 2 seperate endpoints for it)